src/main/java/aquality/appium/mobile/configuration/driversettings/DriverSettings.java

```diff
@@ -14,6 +15,8 @@
     private static final String appPathKey = "applicationPath";
     private static final String appCapabilityKey = "app";
 
+    abstract ISettingsFile getSettingsFile();
```

Comment:
maybe it will be better to do not make this class abstract, but leave as package-private
it will allow to re-use constructor (that currently the same for Android and iOS) and getSettingsFile method